src/helpers/strategies.ts

```diff
@@ -12,6 +12,13 @@ export default Object.fromEntries(
       const key = fileName.replace('./', '').replace('/index.ts', '');
       const strategy = requireFile(fileName);
       strategy.key = key;
+      try {
+        strategy.description = requireFile(
```

Comment:
Can you rename it to `about`
```suggestion
        strategy.about = requireFile(
```